src/components/select/select.spec.js

```diff
@@ -152,7 +153,11 @@ describe('<md-select>', function() {
     it('should not trigger ng-change without a change when using trackBy', function() {
       var changed = false;
       $rootScope.onChange = function() { changed = true; };
-      $rootScope.val = { id: 1, name: 'Bob' };
+
+      // since we're tracking by id, ng-change shouldn't be triggered
+      // when we have two objects that are not strictly equivalent (one has a 'randomAddedProperty')
+      // but that have the same id
```

Comment:
Let's go with "but that have the same tracked field." Since developers may not always be tracking by the `id` field.